packages/react-ui/src/localization/index.js

```diff
@@ -0,0 +1,7 @@
+import en from './en';
+
+const enFlat = en;
```

Comment:
what value does it bring re-assigning `en` to `enFlat`?

why not:

```
export const messages = {
  en
};
```

Reply:
I would probably expect here having the 'es' variations also included

Reply:
A misstep on my part. Before, the flatten was applied there, I forgot to delete it. I'll fix it.
thanks